data_steward/tools/create_rdr_snapshot.py

```diff
@@ -1,16 +1,17 @@
 #!/usr/bin/env bash
 # This Script automates the process of generating the rdr_snapshot and apply rdr cleaning rules
 
-from argparse import ArgumentParser
-from datetime import datetime
+# Python standard imports
 import logging
+from argparse import ArgumentParser
 
-from google.cloud import bigquery
+# Third-party imports
+from gcloud.bq import BigQueryClient
 
+# Project level imports
+from utils import auth
 from cdr_cleaner import clean_cdr
 from cdr_cleaner.args_parser import add_kwargs_to_args
```

Comment:
You're missing your third party imports and have a project import listed with the set of third party imports.  I would expect this to look like:
```suggestion
# Third-party imports
from google.cloud.bigquery.job import CopyJobConfig, WriteDisposition

# Project level imports
from cdr_cleaner import clean_cdr
from cdr_cleaner.args_parser import add_kwargs_to_args
from gcloud.bq import BigQueryClient
from utils import auth
```

This explains the recent linting error.